Mathlib/SetTheory/Ordinal/Nimber.lean

```diff
@@ -357,4 +360,209 @@ theorem add_trichotomy {a b c : Nimber} (h : a + b + c ≠ 0) :
   · rw [← hx'] at hx
     exact Or.inr <| Or.inr hx
 
+theorem two_nsmul (a : Nimber) : 2 • a = 0 := by
```

Comment:
I should be using them, I'll fix that shortly.

Reply:
Actually, I can't use them since they require a ring structure, which I'm trying to prove! I've instead made these private and added the more general versions in the `Algebra.CharP.Two` file.